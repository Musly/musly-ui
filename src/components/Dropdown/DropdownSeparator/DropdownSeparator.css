.dropdown__separator {
  @apply my-2 border-b border-gray-100;
}
