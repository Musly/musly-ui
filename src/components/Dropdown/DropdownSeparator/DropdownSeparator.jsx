import React from 'react';
import './DropdownSeparator.css';

export function DropdownSeparator () {
  return (
    <li className="dropdown__separator" />
  );
}
