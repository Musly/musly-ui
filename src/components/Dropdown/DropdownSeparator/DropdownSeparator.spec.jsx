import React from 'react';
import { shallow } from 'enzyme';
import { DropdownSeparator } from './DropdownSeparator';

describe('<DropdownSeparator />', () => {
  it('should render as expected', () => {
    const result = shallow(<DropdownSeparator />);
    expect(result).toMatchSnapshot();
  });
});
