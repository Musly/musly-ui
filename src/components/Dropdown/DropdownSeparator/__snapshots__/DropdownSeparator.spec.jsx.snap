// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<DropdownSeparator /> should render as expected 1`] = `
<li
  className="dropdown__separator"
/>
`;
